### PR TITLE
questions about l&r edges

### PR DESCRIPTION
**Pull request by edwarkentin**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,9 +6,25 @@
...
     <title>Classwork/Practice</title>
     <link rel="stylesheet" href="css/bootstrap.min.css">    
-    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="css/styles.css">
 </head>
 <body>
-
+<div class="box container-fluid col-md-12 row yellow">[container-fluid]
+    <div class="container">
+        <div class="box header container col-md-12 row blue">[header]</div>
```
Comment (ayojackay) [inline index.html]: Once you call in container as a parent, the rest of the divs don’t need it as long as their within the container. So in your case, I would remove all the class `containers` where you call in the `col-md-etc`

File: index.html [hunk 2]
```diff
@@ -6,9 +6,25 @@
...
+        <div class="box nav container col-md-12 row green">[nav]</div>
+        <div class="row">
+            <div class="box content container col-md-9 red">[content]</div>
+            <div class="box aside container col-md-3 light-blue">[aside]</div>
+        </div> <!--Why does it poke out on the right?-->
+        <div>
```
Comment (ayojackay) [inline index.html]: Try adding a class of `row` here in this empty div.

File: index.html [hunk 3]
```diff
@@ -6,9 +6,25 @@
...
+            <div class="box promo container col-md-3 orange">[promo 1]</div>
+            <div class="box promo container col-md-3 orange">[promo 2]</div>
+            <div class="box promo container col-md-3 orange">[promo 3]</div>
+            <div class="box promo container col-md-3 orange">[promo 4]</div>
+        </div>
+        <div class="box footer container col-md-12 black">[footer]</div> <!--Why do the orange and black divs appear 'indented'?-->
```
Comment (ayojackay) [inline index.html]: I would wrap this footer div in a `<div class=“row”>` too

File: index.html (hunk 2 above)
Comment (edwarkentin) [reply index.html]: It already has one. My comment is at the closing tag of the div, but I have "row" at the opening tag of it.

Comment (ayojackay) [reply index.html]: Ahh yes I see that. I’ll give you an example of what I mean.

`<div class=“container”>`
       `<div class=“row”>`
              `<div class=“col-md-12 box some-other-class”>`
                       (Some content)
                `</div>`
           `</div>`
  `</div>`

Comment (ayojackay) [reply index.html]: Dang it I was hoping it would be structured correctly but it didn’t take lol. But I hope you get what I mean!

Comment (edwarkentin) [reply index.html]: So you mean a container "contains" a row, and a row "contains" col's in increasingly nested divs. Is that correct?

Comment (edwarkentin) [reply index.html]: Whoops - apparently you can't paste code in here, or it renders it, instead of displaying it. Ha!

Comment (ayojackay) [reply index.html]: You can actually do it in your editor on the same branch. Git add., git etc... and will update the pr with the newest commit and show up here . It’s super cool.

Comment (edwarkentin) [reply index.html]: I just did that (yes, SUPER cool), but please don't worry about replying right now. I have to get to bed, and I'm sure you do too... :)

File: index.html [hunk 4]
```diff
@@ -6,9 +6,25 @@
...
 <body>
-
+<div class="box container-fluid col-md-12 row yellow">[container-fluid]
+    <div class="container">
+        <div class="box header container col-md-12 row blue">[header]</div>
+        <div class="box nav container col-md-12 row green">[nav]</div>
+        <div class="row">
+            <div class="box content container col-md-9 red">[content]</div>
+            <div class="box aside container col-md-3 light-blue">[aside]</div>
+        </div> <!--Why does it poke out on the right?-->
```
Comment (ayojackay) [inline index.html]: hey ed etc etc code

**Review by ayojackay — CHANGES_REQUESTED** [review]

**Review by ayojackay — APPROVED** [review]

**Review by ayojackay — APPROVED** [review]

**Pull request merged**